### DISCUSSION TITLE
北京理工大学（良乡）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#31] [University]
**宿舍是上床下桌吗**
是

**教室和宿舍有没有空调**
是

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
有

**有早自习、晚自习吗**
看情况，有的会有

**有晨跑吗**
早上拍照打卡

**每学期跑步打卡的要求是多少公里，可以骑车吗**
72km 36次 甚至可以gps模拟

**寒暑假放多久，每年小学期有多长**
短 2周？

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
可以 大概下楼边上即可

**学校附近有地铁站吗**
不是很便利 有 且有摆渡车 不在 校车1h

**宿舍楼有洗衣机吗**
有

**校园网怎么样**
不限量但限速

**每天断电断网吗，几点开始断**
断电不断网 夏天/考试周不断电 不然0点断

**食堂价格贵吗，会吃出异物吗**
还可以 小概率

**洗澡热水供应时间**
4：30-11：30

**校园内可以骑电瓶车吗，电池在哪能充电**
不可以

**宿舍限电情况**
大功率跳闸

**通宵自习有去处吗**
理论上有，实际没试过

**大一能带电脑吗**
我反正没被禁止，就算真没电脑请前往徐特立网吧（bushi

**Comment by YDX-2147483647** [comment]
> **有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
> 有

静园没有独立卫浴，而是每层两端有浴室、卫生间，并且浴室有隔间。

**Comment by YDX-2147483647** [comment]
> **有早自习、晚自习吗**
> 看情况，有的会有

这取决于班委。

**Comment by YDX-2147483647** [comment]
> **有晨跑吗**
> 早上拍照打卡

没有。

**Comment by YDX-2147483647** [comment]
> **大一能带电脑吗**
> 我反正没被禁止，就算真没电脑请前往徐特立网吧（bushi

基本全校同学都要学习C语言，而这门课要求自带电脑。（据老师说，很久以前是在机房上，但发现自带电脑的人太多，就改到普通教室了。）
